app/talk/popular-tags.cjsx

```diff
@@ -23,6 +26,7 @@ module?.exports = React.createClass
     talkClient.type('tags/popular').get query
 
   tag: (talkTag, i) ->
+    logClick = @context.geordi?.makeHandler? 'hashtag-sidebar'
```

Comment:
This is being used. In an earlier commit, the hashtag call was missing `data` for a project-specific talk. The newest commit resolves this.